### DISCUSSION TITLE
Cannot change RLS Insert policy check

### DISCUSSION THREAD

**Discussion by Nibroc00** [supabase/supabase#28198] [Questions]
I was testing some of my RLS policies but I kept getting policy violations for one specific INSERT. The policy was supposed to allow only authenticated users to insert into the table. When I went to inspect the policy no amount of trying to change the check was persisting after I hit save. Am I missing something here, a small setting perhaps?

Attached is a video showcasing the issue:
https://github.com/user-attachments/assets/40d1df7c-c4bb-4ca7-8d46-30b342417f3a

**Comment by GaryAustin1** [comment]
Your link does not work.  What is the policy you have right now that is getting the error?

**Reply by Nibroc00** [reply]
Link is updated and should work now and shows the policy

**Reply by GaryAustin1** [reply]
It is like you have some sort of error you are not seeing maybe.  Try a different browser?

Is this hosted or local dev?

**Reply by Nibroc00** [reply]
local dev. I've tried it with supabase cli version 1.187.3 and the latest 1.187.6. The video is using Firefox, I just tried it with Chrome and Edge with the same result.

**Reply by Nibroc00** [reply]
Now this may be fixable with deleting the policy and creating a new one, but I don't want to lose this instance where it may be highlighting a bug

**Reply by GaryAustin1** [reply]
You don't show your whole screen.  I assume you click save and do you get the green message in upper right that it has been saved?

I skimmed the issues for the dashboard and see nothing like this.

And other policies in this table can be changed?

**Reply by Nibroc00** [reply]
Yes, I did click the save button but no green message appears. All other policies (including insert policies) can be changed,

**Reply by GaryAustin1** [reply]
I guess generate an issue supabase/supabase or contact support.  Does the green message appear for the other policies?  There have been cases like with the Brave browser where the error popup does not show because of an ad block setting.   I assume that would also impact the green screen but don't know for sure.

**Reply by Nibroc00** [reply]
Just tested with adblock disabled with the same result. I can create and issue and see what comes from that

**Reply by GaryAustin1** [reply]
You might also check the Postgres logs and see if there was an error.  It is odd if the green message is not showing and no error is.

**Reply by Nibroc00** [reply]
Funny enough the postgres logs are not working on Supabase Studio. Waiting for a fix as listed in this issue below.

https://github.com/supabase/supabase/issues/26785
